lib/timers.js

```diff
@@ -87,13 +87,20 @@ function listOnTimeout() {
 
   var first;
   while (first = L.peek(list)) {
-    // If the previous iteration caused a timer to be added,
```

Comment:
We want to keep these comments because they are still relevant: we still need to update `now` or use `Timer.now()` to make sure timing computations are done correctly. However we may want to move these comments to were we use `Timer.now()` right below.